### DISCUSSION TITLE
支付微信支付V3吗？

### DISCUSSION THREAD

**Discussion by Alex37882388** [w7corp/easywechat#2352] [Q&A]
非常好用，但是，支付V3支付吗？

**Comment by overtrue** [comment]
6.x 开始支持

**Comment by anhao** [comment]
v3 有几个接口还是得用v2

**Reply by qypt15** [reply]
"Class \"EasyWeChat\\Pay\\Application\" not found

use EasyWeChat\Pay\Application;   不存在啊？ 请问这个怎么弄

**Reply by anhao** [reply]
有导入包？ 
composer require w7corp/easywechat #下载依赖
require_one ".../vendor/autoload.php" #实际authload 文件地址
